src/encoded/static/components/genome_browser.js

```diff
@@ -703,23 +703,28 @@ class GenomeBrowser extends React.Component {
     /* eslint-disable react/no-did-update-set-state */
     componentDidUpdate(prevProps, prevState) {
         if (!(this.state.disableBrowserForIE) && this.GV) {
-            if (this.state.contig !== prevState.contig) {
+            const contigUpdate = this.state.contig !== prevState.contig;
```

Comment:
`contigUpdate` follows react-naming style of booleans so it is fine. But personally I prefer `IsContigUpdated`. WIth that, I know it is a boolean and would not be expected a value like a number or string.